src/modules/Kahoot/kahoot.controller.ts

```diff
@@ -61,39 +71,21 @@ class KahootController extends CrudController implements Controller {
         .populate('questions')
         .lean();
       if (!data) {
-        return HttpResponse(
-          res,
-          {
-            message: `${id} not found`
-          },
-          status.NOT_FOUND
-        );
+        return Exceptions.NotFound(res, id);
       }
       return HttpResponse(res, { data });
     } catch (error) {
-      return HttpResponse(
-        res,
-        { error: error.message },
-        status.INTERNAL_SERVER_ERROR
-      );
+      return Exceptions.ServerError(res, error);
```

Comment:
Anh vừa update package, em đổi toàn bộ Exceptions thành Exception. Em vô search trên leftbar của vscode nó có chức năng thay đổi toàn bộ đó.